docs/sql-guide/expressions/expressions-home.md

```diff
@@ -1,42 +1,80 @@
 ---
-title: SQL Expressions
+title: Expressions
 layout: default
 parent: SQL guide
-has_children: true
-nav_order: 5
-has_toc: false
 ---
 
 # SQL Expressions
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-db-create-before-begin.md %}
 
 ## identifier
 ![expr](/assets/images/sql-guide/identifier.svg)
 
+{% include /concepts/standard-naming-obj.md %}
+
 ## expr
+
 ![expr](/assets/images/sql-guide/expr.svg)
 
-## integer_literal
+| Syntax | Example | Result |
+|---|---|---|
+| `expr [NOT] LIKE expr` | `SELECT * from Products where product_name LIKE pen` | `ballpen`, `pencil`, `playpen` |
+| `expr IS [NOT] null` | `SELECT * from Services where Price is null` | `free delivery on orders over $50` |
+| `expr [NOT] between expr AND expr` | `SELECT * from Products WHERE Price BETWEEN 10 AND 20;` | `fountain pen`, `notepad`, `rubber ball` |
+| `expr [NOT] IN ([SELECT_stmnt | expr,...])` | `Select * from Products where product_ID NOT IN (SELECT * from Sales where product_name LIKE ear)` | `gold-plated earplugs` |
+
+### Literals
+
+Literals are explicitly specified fixed values that conform to a specific data type:
+
+| String literal | BNF diagram | Further information |
```

Comment:
> I like the diagrams in the table! I'm not sure I understand it though.

This was me trying to create order from what I found in the original doc -- http://docs.featurebase.com/sql-preview/sql-operators-and-expressions#expr

Rather than having a heading for each of the `literal` expressions I thought it would be clearer to whack them all in a table.

Now if the literals end up having exactly the same syntax, then the diagram column can be ripped out, the BNF added between the `### Literals` heading and the shortened table and everyone goes home smiling. Hopefully.

> where did tuple come from?

Tuple was in the original doc with a diagram but no definition. But you knew that didn't you? :)

I confess to having cheated with the definition and got it from here: https://www.w3schools.com/python/python_tuples.asp

In hindsight, I should've asked if this was correct before adding it. My bad >.<